go-controller/go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/mitchellh/copystructure v1.2.0
 	github.com/onsi/ginkgo v1.15.0
 	github.com/onsi/gomega v1.10.1
-	github.com/ovn-org/libovsdb v0.5.1-0.20210708171313-d17ab7f31fc4
+	github.com/ovn-org/libovsdb v0.5.1-0.20210727164307-08dabaa32c8c
```

Comment:
heads up: https://github.com/ovn-org/ovn-kubernetes/pull/2369 bump's this further with fixes for update and mutate insert ops.

Reply:
fixed, increased the version that I bumped the api too